server/modules/auth/repository/jwtRepository.js

```diff
@@ -0,0 +1,25 @@
+const db = require('../../../db/db')
+
+const blacklistToken = async (token, expiration, client = db) =>
+  await client.query(`
+    INSERT INTO jwt_token_blacklist (token, expiration)
+    VALUES ($1, $2)
+  `, [token, expiration])
```

Comment:
expiration is already in token. Could this method take in input the token only and the parameters array would become ```[token, Jwt.getExpiration]``` ?

Reply:
Moved Jwt.getExpiration to authService as agreed verbally

Reply:
maybe Jwt.getExpiration is extracted from the service module as discussed?